### ISSUE TITLE
A11y_.NET Core_MAUI_Recipe App_Screen reader : Narrator is announcing button but there is no visible control/button present in the screen

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/maui#15598]
### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[MAUI]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#Benchmark;#Narrator;#NVDA;

### Actual:
Narrator screen reader focus is going to invisible control, and it is announcing as button.

Note : This bug is also reproducing with NVDA Screen reader.

### Expected:
The Narrator focus should not go to the Invisible content and announce the Control name OR the Controls should be made visible so that Users can see and hear the announcement as well.

### User Impact:
Low vision and Screen reader users will get confused about the control that it is visually present, or it is a Screen reader issue.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Tab till "search box" and enter recipe name and search.
3. "Search result" page will open.
4. Tab till any recipe and select.
5. "Recipe" Screen will open.
6. Turn on Narrator and Verify that the bug by navigating from top of the page.

### Attachment:

Narrator:
[Recording-20230607_205431.webm](https://github.com/dotnet/maui/assets/85863895/f531732c-6e9e-46b7-85c6-6b0a96c01462)

NVDA :
[Recording-20230614_201612.webm](https://github.com/dotnet/maui/assets/85863895/555edf0b-e19d-4e92-8441-72ce43388f7c)

### Link to public reproduction project repository

Test

### Version with bug

.NET 8.0

### Last version that worked well

6.0

### Affected platforms

Windows

### Affected platform versions

Windows 11 23H2

**Comment by rachelkang** [comment]
This is a platform issue (external), but we have a workaround fix coming in https://github.com/dotnet/maui/pull/15835

**Comment by merriemcgaw** [comment]
@Ashutoshsahoo164 can you verify the workaround that @rachelkang mentions and approve closing this issue?

**Comment by rachelkang** [comment]
The fix has been included in the app. Please pull down the latest version of https://github.com/rachelkang/recipesearch and use the same instructions previously provided to retest this issue!

**Comment by PoojaNamde** [comment]
#A11ypreview;

Added A11ypreview tag

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-14-23;
Verified this issue in the latest env and issue does not repro hence closing the bug. Please find closing attachment for ref.

[Video 9-14 at 16.50.webm](https://github.com/dotnet/maui/assets/90901519/28820090-7edf-4344-8b45-033e2e8018fe)

**Comment by PoojaNamde** [comment]
@rachelkang We are not getting option to close this bug. Could you please us to close this bug.

**Comment by rachelkang** [comment]
Closing now, thank you!
